website/docs/cdktf/concepts/fundamentals/providers.html.md

```diff
@@ -223,3 +229,129 @@ Terraform supports using local providers. Terraform has to find these providers
 
 Once configured properly, you can reference these providers in the `cdktf.json` config file the same way that you reference providers in the Terraform Registry.
 
+
+## Resources
+
+Resources are the most important element when defining infrastructure in CDKTF applications. Each resource describes one or more infrastructure objects, such as virtual networks, compute instances, or higher-level components such as DNS records.
+
+### Define resources
+
+Resource definitions and properties vary depending on the type of resource and the provider. Consult your provider's documentation for a full list of available resources and their configuration options.
+
+The TypeScript example below defines a [DynamoDB table](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dynamodb_table) resource on the AWS provider.
+
+``` typescript
+export class HelloTerra extends TerraformStack {
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    new AwsProvider(this, "aws", {
+      region: "eu-central-1",
+    });
+
+    const region = new DataAwsRegion(this, "region");
+
+    const table = new DynamodbTable(this, "Hello", {
+      name: `my-first-table-${region.name}`,
+      hashKey: "temp",
+      attribute: [{ name: "id", type: "S" }],
+      billingMode: "PAY_PER_REQUEST",
+    });
+
+```
+
+The [Examples](/docs/cdktf/examples.html) page contains multiple example projects for every supported programming language. Add something about how to define resources.
+
+### Import Existing Resources
+
+TODO: Add something about how you import existing resources. Please also provide an example :-)
```

Comment:
There's no explicit import support in cdktf, but this can be done by using TF Core. However, this depends on which backend is used and is a bit fiddly. Do we want to put it there anyway?

Reply:
okay - thank you for the update! Let's omit then for now :-) If there's no explicit support, then I don't think we should talk about it.